integrations/Mouseflow/browser.js

```diff
@@ -34,33 +34,33 @@ class Mouseflow {
    * Identify.
    * for supporting userId or email
    * Ref: https://js-api-docs.mouseflow.com/#identifying-a-user
-   * for supporting user traits and tags
+   * for supporting user traits and customVariables
    * Ref: https://js-api-docs.mouseflow.com/#setting-a-custom-variable
    * @param {Identify} identify
    */
   identify(rudderElement) {
+    logger.debug("===In mouseflow Identify===");
     const { message } = rudderElement;
     const { traits } = message.context;
     const email = message.context.traits?.email || message.traits?.email;
-    const userId = message.userId || email;
+    const userId = message.userId || email || message.anonymousId;
     window._mfq.push(["stop"]);
-    if (userId) window._mfq.push(["identify", userId]);
+    if (userId) window.mouseflow.identify(userId);
```

Comment:
Why use the object instead of the queue here?

Reply:
As with queue, we tried but it does not work, and by using object it works, so we take this.

Reply:
It is a weird behavior of their SDK. They can't run the function if we use the queue after stopping their SDK. So we have to use the object.

Reply:
You are doing it wrong. Identify the user after starting the recording.

Reply:
Actually, why are we stopping and starting the recording in the first place?

Reply:
Sorry my bad, I'll update it